modules/ephemeris/src/test/scala/gem/horizons/HorizonsEphemerisQueryPagingPropSpec.scala

```diff
@@ -83,7 +80,7 @@ class HorizonsEphemerisQueryPagingPropSpec extends PropSpec with PropertyChecks
     }
   }
 
-  property("each element is spaced exactly by duration") {
+  test("each element is spaced exactly by duration") {
```

Comment:
I don't recall `property` was it related to scalacheck?

Reply:
Yeah you can write tests with straight scalacheck and no scalatest.